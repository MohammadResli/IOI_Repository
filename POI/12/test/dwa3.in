30
22 5 19 28 10 18 27 4 11 16 7 12 21 17 23 22 29 30 15 13 20 24 25
23 23 6 16 17 15 3 24 5 8 4 25 22 14 30 12 9 28 13 19 21 7 29 11
22 26 20 30 7 17 24 15 21 12 4 25 16 10 14 2 29 8 23 28 22 11 13
21 16 11 22 7 29 13 12 23 9 25 3 24 1 17 14 15 28 2 20 8 26
20 12 14 1 26 24 2 21 7 23 6 18 15 8 29 19 16 17 11 27 25
24 29 30 26 7 22 10 15 11 25 21 12 20 5 18 28 27 8 14 13 16 17 19 23 2
25 22 11 26 3 9 14 29 8 27 25 23 20 4 28 5 16 19 18 13 6 1 2 17 12 30
26 2 22 21 27 23 5 14 9 7 30 4 3 28 15 26 17 18 6 11 10 25 19 29 12 13 24
20 22 23 18 11 30 15 28 7 13 19 29 2 14 16 20 8 12 24 10 4
20 18 27 26 13 22 16 29 21 19 12 6 9 30 8 23 14 1 24 3 25
28 17 6 27 15 22 9 1 5 23 8 4 14 25 2 24 30 28 18 12 26 19 7 21 29 13 3 20 16
26 9 14 18 26 10 2 5 24 19 7 17 25 11 21 15 23 16 27 1 6 28 20 3 8 4 30
19 14 29 22 17 3 25 16 1 11 9 23 2 8 24 7 10 4 28 6
25 24 10 7 18 5 30 21 4 28 12 23 13 11 20 2 27 22 8 15 3 19 9 25 6 29
24 6 30 14 26 18 4 21 17 16 11 3 9 27 24 25 1 8 23 2 29 22 12 5 19
26 27 21 23 4 3 9 30 15 20 26 7 1 29 17 22 13 19 28 10 18 11 5 2 24 6 12
20 15 12 3 26 8 7 13 11 5 30 4 16 28 25 23 6 2 1 20 24
19 16 19 6 24 26 9 30 15 28 11 12 21 14 29 10 8 1 7 5
24 27 22 10 24 16 1 11 12 7 6 8 14 20 25 15 2 9 5 18 23 28 26 29 30
20 4 9 27 26 6 28 30 21 12 14 25 7 1 29 3 17 11 19 24 16
22 18 25 30 27 26 22 10 6 3 5 12 16 1 23 20 8 11 2 29 24 15 14
22 7 10 2 8 9 15 21 26 23 6 28 3 24 27 19 30 14 16 4 1 11 13
27 6 14 24 7 11 28 2 12 27 8 9 19 10 25 16 26 17 15 5 22 4 1 30 13 21 29 3
26 22 11 27 3 5 12 30 8 14 20 13 17 9 1 16 4 26 10 23 19 21 15 2 28 18 29
22 27 14 1 4 12 21 6 15 11 10 13 3 23 19 7 20 8 2 30 26 5 17
23 10 30 17 18 19 24 7 5 23 15 21 3 16 11 28 22 6 8 12 20 29 25 4
19 1 15 20 6 16 24 11 21 19 5 8 25 29 7 10 12 23 14 22
22 3 18 14 11 22 16 1 12 30 4 9 19 8 23 26 7 2 17 20 13 24 6
24 18 13 19 1 14 30 6 20 24 7 21 27 8 3 15 5 11 10 16 2 26 9 4 23
25 20 9 7 14 19 28 15 25 24 16 21 1 23 10 18 8 11 17 2 29 26 22 3 12 6
