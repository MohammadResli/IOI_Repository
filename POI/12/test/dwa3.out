15
1 5 6 8 9 10 13 14 16 19 21 22 28 29 30 
